v1.1.0_release_notes.md

```diff
@@ -0,0 +1,59 @@
+## Breaking Changes.
+dcrpool v1.1.0 introduces breaking changes to the configuration, existing configurations will have to be updated accordingly. The breaking 
+changes made are:
+ 1. `--backuppass` has been renamed to `--adminpass`.
+
+ 2.  `--lastnperiod` and `--maxgentime` are now of type `time.Duration`.
+
+ 3. Added `--walletaccount` config option.
```

Comment:
Not a breaking change. It defaults to 0 if not provided. Suggest moving down to "Improvements", similar to `--maxconnectionsperhost` config